miqa/core/rest/session.py

```diff
@@ -140,5 +140,10 @@ def import_(self, request, **kwargs):
     @action(detail=True, methods=['POST'])
     def export(self, request, **kwargs):
         session: Session = self.get_object()
-        export_data(request.user, session)
-        return Response(status=status.HTTP_204_NO_CONTENT)
+        try:
+            export_data(request.user, session)
+            return Response(status=status.HTTP_204_NO_CONTENT)
+        except IsADirectoryError:
+            return Response({'message': 'Invalid export path'}, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
```suggestion
            return Response({'detail': 'Invalid export path'}, status=status.HTTP_400_BAD_REQUEST)
```
I just learned the other day that DRF has a standard shape for exceptions: https://www.django-rest-framework.org/api-guide/exceptions/#custom-exception-handling